parm/archive/gfs_netcdfa.yaml.j2

```
@@ -6,7 +6,7 @@ gfs_netcdfa:
         - "{{ COMIN_ATMOS_ANALYSIS | relpath(ROTDIR) }}/{{ head }}atmanl.nc"
         - "{{ COMIN_ATMOS_ANALYSIS | relpath(ROTDIR) }}/{{ head }}sfcanl.nc"
         - "{{ COMIN_ATMOS_ANALYSIS | relpath(ROTDIR) }}/{{ head }}atminc.nc"
-        {% for iauhr in iaufhrs if iauhr != 6 %}
+        {% for iauhr in IAUFHRS if iauhr != 6 %}
```

For 3DVar, `IAUFHRS=6`, so it will not be a list when parsed by wxflow.  Instead, you may need an additional check here:

```suggestion
        {% if IAUFHRS is not integer %} # If it is, then there will only be one increment file (handled above)
            {% for iauhr in IAUFHRS if iauhr != 6 %}
```